unfetter-discover/unfetter-ui
# initial file structure and components for threat dashboard refresh

fixes unfetter-discover/unfetter#1439

To Test:

- Verify Threat Dashboard Beta in main menu drop-down
- Verify Beta icon
- Verify navigating takes you to a placeholder page with url /threat-beta
- Verify direct navigation to /threat-beta/create takes you to placeholder page
- Verify direct navigation to /threat-beta/feed takes you to placeholder page with tab navigation
- Verify tab navigation on /threat-beta/feed navigates among /board and /article placeholder pages
